src/crimson/os/alienstore/alien_store.cc

```diff
@@ -108,6 +108,9 @@ seastar::future<> AlienStore::start()
   // 	[0, N_CORES_FOR_SEASTAR) are reserved for seastar reactors
   // 	[N_CORES_FOR_SEASTAR, ..] are assigned to alien threads.
   if (!cpu_cores.has_value()) {
+    // number of cores that are PREVENTED from being scheduled
+    // to run alien store threads.
+    int N_CORES_FOR_SEASTAR = get_conf<uint64_t>("crimson_seastar_pinned_cpu_cores");
     seastar::resource::cpuset cpuset;
     std::copy(boost::counting_iterator<unsigned>(N_CORES_FOR_SEASTAR),
 	      boost::counting_iterator<unsigned>(sysconf(_SC_NPROCESSORS_ONLN)),
```

Comment:
Updated vstart to make use of existing crimson_alien_thread_cpu_cores option.
I removed magic 3 and preferred no isolation by default.

Reply:
Seems like we probably want configs for this -- how about we let the user specify a cpuset?  We want to move the machinery for specifying bluestore/alienstore cpuset from the benchmark script into a real config sooner rather than later I'd think.

Reply:
@athanatos do you mean add --crimson_alien_thread_cpu_cores to vstart.sh options? @Matan-B I think better to make alienstore pin to 0-(smp-1) by default.

Reply:
> Seems like we probably want configs for this -- how about we let the user specify a cpuset? 

It is possible using `crimson_alien_thread_cpu_cores`, this PR is for making vstart to make use of this option. Previously it was set to 3 by default. 

> I think better to make alienstore pin to 0-(smp-1) by default.

I'm not sure I understand why, can you explain?
The default vstart will now use [0, `smp * osd`) for seastar and [`smp * osd`, (nproc  - 1)] for the alienstore.

Reply:
Oh, ignore my comment -- it's already a config.

Reply:
@Matan-B when we set smp number, we may think crimson + (cyanstore and seastore) just use that number of cpus but for crimson+alienstore it will use all available cpus in the system, so better bind all threads to the number of same as smp, keep them the same by default.

Reply:
> @Matan-B when we set smp number, we may think crimson + (cyanstore and seastore) just use that number of cpus but for crimson+alienstore it will use all available cpus in the system, so better bind all threads to the number of same as smp, keep them the same by default.

Got it. I agree that alienstore should be (somewhat)  aligned with cyanstore/seastore default behavior.